File: Data Structures & Algorithms/kth-smallest-integer-in-bst/submission-0.java
/**
 * Definition for a binary tree node.
 * public class TreeNode {
 *     int val;
 *     TreeNode left;
 *     TreeNode right;
 *     TreeNode() {}
 *     TreeNode(int val) { this.val = val; }
 *     TreeNode(int val, TreeNode left, TreeNode right) {
 *         this.val = val;
 *         this.left = left;
 *         this.right = right;
 *     }
 * }
 */

class Solution {
    public int kthSmallest(TreeNode root, int k) {
        List<Integer> list = new ArrayList<>();
        getList(list, root);
        
        return list.get(k-1);
    }
    private void getList(List<Integer> list, TreeNode node){
        if(node!=null){
            getList(list, node.left);
            list.add(node.val);
            getList(list, node.right);
        }
    }
}

File: Data Structures & Algorithms/level-order-traversal-of-binary-tree/submission-0.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, val=0, left=None, right=None):
#         self.val = val
#         self.left = left
#         self.right = right

class Solution:
    def levelOrder(self, root: Optional[TreeNode]) -> List[List[int]]:
        ret = []
        if not root:
            return ret
        d = deque()
        d.append(root)

        while d:
            size = len(d)
            level = []
            for i in range(size):
                temp = d.popleft()
                if temp:
                    level.append(temp.val)
                    if temp.left:
                        d.append(temp.left)
                    if temp.right:
                        d.append(temp.right)
            ret.append(level)

        return ret